apiserver/deployment.yaml.in

```diff
@@ -0,0 +1,138 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: spark-master
+spec:
+  selector:
+    matchLabels:
+      app: spark-master
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: spark-master
+    spec:
+      containers:
+        - name: spark-master
+          image: @spark_master_image@
+          imagePullPolicy: Always
+          ports:
+          - containerPort: 7077
+          - containerPort: 8080
+          volumeMounts:
+          - mountPath: /hail-vdc-sa-key
+            name: hail-vdc-sa-key
+          resources:
+            requests:
+              memory: "3.75G"
+              cpu: "0.9"
+      volumes:
+        - name: hail-vdc-sa-key
+          secret:
+            secretName: hail-vdc-sa-key
+---
+apiVersion: apps/v1beta2
```

Comment:
Is there a reason to use v1beta2 over v1?

Reply:
No.  v1 probably wasn't ready yet when we started using deployments and this is copy-pasta.

Reply:
For the record, you can get the list of supported api versions with:

```
$ kubectl api-versions
apps/v1
...
```

apps/v1 is supported by our current cluster.